### PR TITLE
Javascript Capstone Group Project

### PR DESCRIPTION
**Pull request by Phone-Khaing-Hein**
### This is our second capstone project at Microverse. My coding partner and I have managed to accomplish the mentioned tasks in this amazing activity: 

- [x] Linter Configuration
- [x] Webpack Configuration
- [x] Jest Configuration
- [x] Home Page UI
- [x] Fetching Data for the Home page using The Meal DB API
- [x] Comment Modal Box UI
- [x] Fetching description about meals using API
- [x] Create a new comment
- [x] Calculate comments count
- [x] Create Like using API
- [x] Calculate the Likes count
- [x] Calculate Items count
- [x] Items count testing
- [x] Comments count testing

[**We would love to hear from you, you can check the project presentation link here**](https://drive.google.com/file/d/1FLbcPfNVkNzWiEMkb8TQeGMG_-2Duq97/view?usp=sharing)

### CODE REVIEW

File: src/modules/comments.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      item_id: id,
+      username,
+      comment,
+    }),
+  });
```
Comment (divinecharlotte) [inline src/modules/comments.js:46]: - [x] kindly consider making sure that when someone adds a comment the add comment button POST request to the [Involvement API](https://www.notion.so/Involvement-API-869e60b5ad104603aa6db59e08150270) . the data is expected to get recorded in the [Involvement API](https://www.notion.so/Involvement-API-869e60b5ad104603aa6db59e08150270) and the screen is updated.currently when I add a comment the screen is getting updated but nothing is posted to the API cause once I refresh the page the comment disappeared and the count returns back to zero fixing that will make your comments meet the requirement

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @Team , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive PR ✔️ 

## Required Changes ♻️

_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/comments.js (hunk 1 above)
Comment (Phone-Khaing-Hein) [reply src/modules/comments.js:46]: Fixed.

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi Team,

#### Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

### Congratulations! 🎉 

giphy (1)

## (Highlights) Good Points: :+1:
- Your work is well documented. ✔
- The correct workflow is used.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**